tests/pka_test_utils.c

```diff
@@ -4002,6 +4160,53 @@ pka_operand_t *sw_mod_exp_with_crt(pka_handle_t   handle,
     return NULL;
 }
 
+pka_operand_t *sw_mont_ecdh_multiply(pka_handle_t      handle,

```

Comment:
Doesn't look right to me. How a software implementation offloads operations to hardware?